packages/database/src/migrations/20210317035935-AddConfigForCategoryHeadingLaosEoc-modifies-data.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+import { updateValues } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function getDashboardReportById(db, id) {
+  const { rows: dashboardReports } = await db.runSql(`
+      SELECT * FROM "dashboardReport"
+      WHERE id = '${id}';
+  `);
+  return dashboardReports[0] || null;
+}
+
+const categoryPresentationOptions = {
+  conditions: {
+    red: {
+      color: '#b71c1c',
+      legendLabel: 'Stock out',
+    },
+    green: {
+      color: '#33691e',
+      label: '',
+      legendLabel: 'In stock',
+    },
+    orange: {
+      color: 'orange',
+      label: '',
+      legendLabel: 'At least 1 item out of stock',
+    },
+  },
+  showRawValue: true,
+  showNestedRows: true,
+};
+
+const previousPresentationOptions = {
+  type: 'condition',
+  conditions: [
+    {
+      key: 'red',
+      color: '#b71c1c',
+      label: '',
+      condition: 0,
+      description: 'Stock number: ',
+      legendLabel: 'Stock out',
+    },
+    {
+      key: 'green',
+      color: '#33691e',
+      label: '',
+      condition: {
+        '>': 0,
+      },
+      description: 'Stock number: ',
+      legendLabel: 'In stock',
+    },
+  ],
+  showRawValue: true,
+};
+
+const newPresentationOptions = {
+  type: 'condition',
+  conditions: [
+    {
+      key: 'red',
+      color: '#b71c1c',
+      label: 'Stock number: 0',
+      condition: {
+        '=': '',
+      },
+      legendLabel: 'Stock out',
+    },
+    {
+      key: 'green',
+      color: '#33691e',
+      label: '',
+      condition: {
+        '>': 0,
+      },
+      description: 'Stock number: ',
+      legendLabel: 'In stock',
+    },
+    {
+      key: 'orange',
+      color: 'orange',
+      label: '',
+      legendLabel: 'At least 1 item out of stock',
+    },
+  ],
+  showRawValue: true,
+};
+
+const categoryAggregator = {
+  type: '$condition',
+  conditions: [
+    {
+      key: 'red',
+      condition: {
+        '=': null,
+      },
+    },
+    {
+      key: 'green',
+      condition: {
+        '>': 0,
+      },
+    },
+    {
+      key: 'orange',
+      condition: {
+        some: { '>': 0 },
```

Comment:
I like the simplicity of `some`, but devs may confuse it with `array.some` which is true if all items satisfy the given condition. In fact, this is how I interpreted this config before reading the rest of the code.

Let's use something more specific, like `someNotAll`

Reply:
Brilliant, that's clearer!